docs/installing/config_options.csv

```diff
@@ -0,0 +1,24 @@
+Key,Value(s),Description,Required
+additional-repositories,[ `multilib <https://wiki.archlinux.org/title/Official_repositories#multilib>`_!, `testing <https://wiki.archlinux.org/title/Official_repositories#Testing_repositories>`_ ],Enables one or more of the testing and multilib repositories before proceeding with installation,No
+archinstall-language,`lang <https://github.com/archlinux/archinstall/blob/master/archinstall/locales/languages.json>`_,Sets the TUI language used *(make sure to use the ``lang`` value not the ``abbr``)*,No
+audio_config,`pipewire <https://wiki.archlinux.org/title/PipeWire>`_!, `pulseaudio <https://wiki.archlinux.org/title/PulseAudio>`_,Audioserver to be installed,No
+bootloader,`Systemd-boot <https://wiki.archlinux.org/title/Systemd-boot>`_!, `grub <https://wiki.archlinux.org/title/GRUB>`_,Bootloader to be installed *(grub being mandatory on BIOS machines)*,Yes
+debug,``true``!, ``false``,Enables debug output,No
+disk_config,*`read more under Disk Configuration <#>`_*,Contains the desired disk setup to be used during installation,No
```

Comment:
hi @Torxed, is there more information about disk_config anywhere? 
I'm trying to reverse engineer the parameters from running it guided but I haven't had much success haha I stumbled on this branch/PR but couldn't find anything else about disk_config here, that's a section that will be created/updated in the future? Thanks for your time and the amazing tool!

Reply:
I can make sure to update `disk_config` before anything else :) I'm expecting to be working on this during the weekend quite a bit, but can *try* (no promises) to get `disk_config` done before that a :)

Reply:
I would really appreciate it haha Anything I could do to help move things forward? I can try to help to write parts of the documentation I understand but Im not sure if my help would increase or decrease your productivity haha

Reply:
@alan-barzilay I've submitted a decent first draft of how disk partitioning works and the different modes. You'll find them under `archinstall/docs/_build/html/cli_parameters/config/disk_config.html` after running:

```shell
$ cd archinstall/docs
$ make clean html
```

They'll get pushed to the readthedocs once I'm finished here :)